pkg/config/config.go

```diff
@@ -206,7 +268,94 @@ func New() (*Config, error) {
 	}
 
 	if len(b) != 0 {
-		yaml.Unmarshal(b, c)
+		err = yaml.Unmarshal(b, c)
+		if err != nil {
+			return c, err
+		}
+	}
+
+	accessConfigFilePath := filepath.Join(configPath, AccessConfigFileName)
+	accessConfigFile, err := os.Open(accessConfigFilePath)
+	defer accessConfigFile.Close()
+	if err != nil {
+		log.Infof("AccessConfig file not found")
+
+		if (c.Communications.Slack.Enabled && c.Communications.Slack.Channel != "") || (c.Communications.Mattermost.Enabled && c.Communications.Mattermost.Channel != "") {
+			log.Warn("Configuration provided as used in older version of botkube")
+			log.Warn("Check the updated configuration at: https://www.botkube.io/configuration/")
+			log.Warn("setting default profile values to proceed")
+
+			if c.Communications.Slack.Enabled {
+				c.Communications.Slack.Accessbindings = *DefaultAccessBindings(c.Communications.Slack.Channel, c.Settings.Kubectl)
+			}
+			if c.Communications.Mattermost.Enabled {
+				c.Communications.Mattermost.Accessbindings = *DefaultAccessBindings(c.Communications.Mattermost.Channel, c.Settings.Kubectl)
+			}
+
+			return c, nil
+		} else if !c.Communications.Slack.Enabled && !c.Communications.Mattermost.Enabled {
+			return c, nil
+		} else {
+			return c, err
+		}
+	}
+
+	b, err = ioutil.ReadAll(accessConfigFile)
+	if err != nil {
+		return c, err
+	}
+	profiles := &AllProfiles{}
+	if len(b) != 0 {
+		err = yaml.Unmarshal(b, profiles)
+		if err != nil {
+			return c, err
+		}
+	}
+	// Map right profile's value with config: For slack
+	if c.Communications.Slack.Enabled {
+		for i, AccessBind := range c.Communications.Slack.Accessbindings {
```

Comment:
Please use a lowercase character at the start of the local variable name. Same for all other places